ui/app/AppLayouts/Profile/helpers/ProfileShowcaseModelAdapter.qml

```diff
@@ -0,0 +1,212 @@
+import QtQml 2.15
+
+import StatusQ 0.1
+import StatusQ.Core.Utils 0.1
+
+import SortFilterProxyModel 0.2
+
+QObject {
+    id: root
+
+    // input models
+    property alias communitiesSourceModel: communityRenamingSource.sourceModel
```

Comment:
`communitiesRenamingSource` to follow the same naming!

I'm thinking maybe it's better something like this: `renamedCommunitiesSource`?!

Reply:
> communitiesRenamingSource to follow the same naming!
👍 

> I'm thinking maybe it's better something like this: renamedCommunitiesSource?!

Might be too descriptive. Not sure we need the internal details of what this adapter does in the variable. Could find later on that we need new roles as well.